dali/test/python/test_detection_pipeline.py

```diff
@@ -249,13 +249,13 @@ def define_graph(self):
             saturation=saturation,
             contrast=contrast,
             brightness=brightness,
-            hue=0-hue)
+            hue=hue)
```

Comment:
I would remove this `image_legacy_twisted_cpu` and just use a new color twist.
The above:
```
image_twisted_cpu = self.hsv_cpu(image_ssd_crop, saturation=saturation, hue=hue)
image_twisted_cpu = self.bc_cpu(image_twisted_cpu, brightness=brightness, contrast=contrast)
image_twisted_gpu = self.hsv_gpu(image_ssd_crop.gpu(), saturation=saturation, hue=hue)
image_twisted_gpu = self.bc_gpu(image_twisted_gpu, brightness=brightness, contrast=contrast)
```
may be not need as well.

Reply:
It's not exactly comparing the same operator because it compares Hsv;BrightnessContrast vs ColorTwist and BrightnessContrast op does not share implementation with ColorTwist.

Reply:
I'd remove the hsv/bc in favor of the revamped ColorTwist - it would be more efficient and people _do_ look at tests as a supplement for tutorials. Also, if we have some reference pipeline somewhere outside tests, let's also convert it to ColorTwist. This may be done in a follow-up, but definitely should be done.